content/learn/book/migration-guides/0.7-0.8/_index.md

```diff
@@ -0,0 +1,513 @@
+
+### [Mark mutable APIs under ECS storage as pub(crate)](https://github.com/bevyengine/bevy/pull/5065)
+
+<!-- Dear God, I hope not. -->
+
+If you experienced any problems caused by this change, please [create an issue](https://github.com/bevyengine/bevy/issues) explaining _in detail_ what you were doing with those apis.
+
+### [Add global init and get accessors for all newtyped TaskPools](https://github.com/bevyengine/bevy/pull/2250)
+
+Thread pools don't need to be stored in a resource anymore since they are now stored globally. You can now use `get()` to access it.
+
+```rust
+// 0.7
+fn spawn_tasks(thread_pool: Res<AsyncComputeTaskPool>) {
+    // Do something with thread_pool
+}
+
+// 0.8
+fn spawn_tasks() {
+    let thread_pool = AsyncComputeTaskPool::get();
+    // Do something with thread_pool
+}
+```
+
+### [Simplify design for *Labels](https://github.com/bevyengine/bevy/pull/4957)
+
+* Any previous use of `Box<dyn SystemLabel>` should be replaced with `SystemLabelId`.
+* `AsSystemLabel` trait has been modified.
+  * No more output generics.
+  * Method `as_system_label` now returns `SystemLabelId`, removing an unnecessary level of indirection.
+* If you _need_ a label that is determined at runtime, you can use `Box::leak`. Not recommended.
+
+### [Move get_short_name utility method from bevy_reflect into bevy_utils](https://github.com/bevyengine/bevy/pull/5174)
+
+* added bevy_utils::get_short_name, which strips the path from a type name for convenient display.
+* removed the TypeRegistry::get_short_name method. Use the function in bevy_utils instead.
+
+### [Remove dead SystemLabelMarker struct](https://github.com/bevyengine/bevy/pull/5190)
+
+This struct had no internal use, docs, or intuitable external use.
+
+It has been removed.
+
+### [Add reflection for resources](https://github.com/bevyengine/bevy/pull/5175)
+
+Rename `ReflectComponent::add_component` into `ReflectComponent::insert_component`.
+
+### [Make reflect_partial_eq return more accurate results](https://github.com/bevyengine/bevy/pull/5210)
+
+Updated [struct_trait](https://github.com/bevyengine/bevy/blob/dfe969005264fff54060f9fb148639f80f9cfb29/crates/bevy_reflect/src/struct_trait.rs#L455-L457), [tuple_struct](https://github.com/bevyengine/bevy/blob/dfe969005264fff54060f9fb148639f80f9cfb29/crates/bevy_reflect/src/tuple_struct.rs#L366-L368), [tuple](https://github.com/bevyengine/bevy/blob/dfe969005264fff54060f9fb148639f80f9cfb29/crates/bevy_reflect/src/tuple.rs#L386), [array](https://github.com/bevyengine/bevy/blob/dfe969005264fff54060f9fb148639f80f9cfb29/crates/bevy_reflect/src/array.rs#L335-L337), [list](https://github.com/bevyengine/bevy/blob/dfe969005264fff54060f9fb148639f80f9cfb29/crates/bevy_reflect/src/list.rs#L309-L311) and [map](https://github.com/bevyengine/bevy/blob/dfe969005264fff54060f9fb148639f80f9cfb29/crates/bevy_reflect/src/map.rs#L361-L363) to return `None` when comparison couldn't be performed.
```

Comment:
Previously all reflect `***_partial_eq` helper methods returned `Some(false)` even when the comparison could not be performed, which was misleading. Now those helper methods will return `None` if the comparison could not be performance, while still returning either `Some(true)` or `Some(false)` based on the equality comparison.

Reply:
How about this:
```markdown
Previously, all reflect `***_partial_eq` helper methods returned `Some(false)` when the comparison could not be performed, which was misleading. They now return `None` when the comparison cannot be performed.
```

Reply:
That seems like a huge improvement and the English is perfect. Thanks!

Reply:
If you like it, could you make a code suggestion with the changes. That way you will be credited as a co-author.

Reply:
You deserve to be the co-author, this summary is way cooler 😄. I'm glad to had whenever I can